libs/components/router/src/lib/modules/recently-accessed/recently-accessed.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+
+import { Observable, of } from 'rxjs';
```

Comment:
`of` is not used and should not be here.

Reply:
Ah, we didn't turn on `noUnusedLocals` in `5.x.x`.

Reply:
Good catch! Fixed.